src/huggingface_hub/README.md

```diff
@@ -81,6 +40,29 @@ Parameters:
 - a `cache_dir` which you can specify if you want to control where on disk the
   files are cached
 
+### `hf_hub_url`
+
+Internally, the library uses `hf_hub_url()` to return the URL to download the actual files:
+`https://huggingface.co/julien-c/EsperBERTo-small/resolve/main/pytorch_model.bin`
+
+You're welcome to also use it if you need it.
+
+Parameters:
+- a `repo_id` (a user or organization name and a repo name seperated by a `/`, like `julien-c/EsperBERTo-small`)
+- a `filename` (like `pytorch_model.bin`)
+- an optional `subfolder`, corresponding to a folder inside the model repo
+- an optional `repo_type`, such as `dataset` or `space`
+- an optional Git revision id (can be a branch name, a tag, or a commit hash)
```

Comment:
Do we need this here? The parameters are already listed on the package reference page IIRC

Reply:
Yes I would mention that all arguments are visible in the reference that is linked above, which is more complete

Reply:
well this is not rendered in the doc, just on GitHub